sunrise-bay-area/sunrise-bay-area
# Separate theme

For https://github.com/sunrise-bay-area/sunrise-bay-area/issues/41

Currently moves main index page & layout into a theme, as well as the CMS!

## QA
- Make sure the site still appears as normal
- Make sure netlify cms still works

## Future work
- Extract theme to separate repo
- Implement https://github.com/sunrise-bay-area/sunrise-bay-area/issues/16
- Separate out reusable components e.g. footer to partials
